pyproject.toml

```diff
@@ -162,9 +162,26 @@ source = ["scanpy"]
 omit = ["*/tests/*"]
 
 [tool.black]
-line-length = 88
-target-version = ["py38"]
 skip-string-normalization = true
-exclude = """
-/build/.*
-"""
```

Comment:
- `line-length = 88` is default
- `target-version` is inferred from `requires-python`
- `exclude` is derived from .gitignore